modules/mobile/docs/openapi.yaml

```diff
@@ -19,11 +19,67 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Message"
+  /v0/military-service-history:
+    get:
+      summary: /v0/military-service-history
+      description: Returns user's service history
+      operationId: getServiceHistory
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/MilitaryHistory"
+
 components:
   schemas:
     Message:
       required:
-        - message
+        - attributes
+      properties:
+        attributes:
+          type: object
+          required:
+            - message
+          properties:
+            message:
+              type: string
+              example: "Welcome to the mobile API"
+    MilitaryHistory:
+      required:
+        - data
+      properties:
+        data:
+          required:
+            - type
+            - id
+            - attributes
+          type: array
+          items:
+            type: object
+            properties:
+              type:
+                type: string
+                example: "serviceHistory"
+              id:
+                type: string
+                example: "1"
+              attributes:
+                type: object
+                $ref: "#/components/schemas/ServiceHistory"
+    ServiceHistory:
+      required:
+        - branchOfService
+        - beginDate
+        - endDate
       properties:
-        message:
-          type: string
\ No newline at end of file
+        branchOfService:
+          type: string
+          example: "United States Marine Corps"
+        beginDate:
+          type: string
+          example: "September 17, 1997"
+        endDate:
+          type: string
+          example: "December 31, 2002"
```

Comment:
Should we document our decision that dates are not to be parsed?

Reply:
I think that's a good idea. Do you have thoughts on where that documentation would live? In the description for this endpoint? Or maybe the description field for the entire API?

Reply:
The more I think about this there's a reason ISO exists and we may be loading up a foot gun by pre-formatting dates. I think we should standardize on that.

Reply:
Sure, just to be clear on what will be happening, the current service history endpoint returns the dates in YYYY-MM-DD format, but the website displays them in the [Month] DD, YYYY format. In order to avoid having to do that translation in the app, and avoid needing to release new app versions for fixing/changing things like that, pre-formatting display strings can be useful. Let me know how/where I should go about documenting this decision, as long as everyone is on board with it, and I'll get on that asap

Reply:
Sorry for the back and forth on this @jperk51 took a look at vets-api and lighthouse and they (for the most part) use ISO8601 date format so I think we should stick to that (profile uses ISO date exclusively). I understand wanting to keep as much business logic, including formatting, server side for the mobile app but in this case I think we're covered by the design system/content rules always using [Month] DD, YYYY so that's unlikely to change. Once we get to things like appointment times or other dates that need to be updated it will be easier to know that they're always ISO8601 when coming and going through the API. Please update the example to be YYYY-MM-DD.